ci/ci.sh

```diff
@@ -83,7 +83,19 @@ esac
     --host="$HOST" $EXTRAFLAGS
 
 # We have set "-j<n>" in MAKEFLAGS.
-make
+build_exit_code=0
+make || build_exit_code=$?
+if [ $build_exit_code -ne 0 ]; then
+    case $CC in
+        *snapshot*)
+            make 2>&1 >/dev/null | grep "internal compiler error:"
```

Comment:
Regarding `make 2>&1 >/dev/null`:

I think it would be more reliable to run `make` only once. (Perhaps gcc-snapshot crashes in one run but not in another.)

One way could be: Change the first `make` to `make | tee make.log` and then use `grep` on `make.log`.

Reply:
Thanks! [Done](https://github.com/bitcoin-core/secp256k1/pull/1437#issuecomment-1798325434).